gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSub.java

```diff
@@ -387,25 +387,107 @@ public static PullOption maxConcurrentCallbacks(int maxConcurrency) {
 
   MessageConsumer pullAsync(String subscription, MessageProcessor callback, PullOption... options);
 
+  /**
+   * Acknowledges the given messages for the provided subscription. Ack ids identify the messages to
+   * acknowledge, as returned in {@link ReceivedMessage#ackId()} by {@link #pull(String, int)} and
+   * {@link #pullAsync(String, int)}.
+   *
+   * @param subscription the subscription whose messages must be acknowledged
```

Comment:
I'm new to this (for this repo), so take these with a grain of salt, but looking at other javadoc, this seems overly wordy. @jgeewax @aozarov (please agree or disagree -- I see earlier examples that contradict my comments, many other published javadocs are as I describe.)

The initial "sentence" [should be a fragment](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment).
`Acknowledges given messages for provided subscription.` ...
```
@param subscription whose messages are being acknowledged
@param ackId first message to acknowledge
```